products/cakephp.md

```diff
@@ -232,11 +230,26 @@ releases:
 
 ---
 
-> [CakePHP](https://cakephp.org/) is a free & open source PHP web development framework.  It follows
-> the model–view–controller (MVC) approach and is written in PHP, modeled after the concepts of Ruby
-> on Rails, and distributed under the [MIT License](https://en.wikipedia.org/wiki/MIT_License).
+> [CakePHP](https://cakephp.org/) is a free and open source PHP web development framework. It
+> follows the model–view–controller (MVC) approach and is written in PHP, modeled after the concepts
+> of Ruby on Rails.
 
-Have a look at the [Bakery](https://bakery.cakephp.org/) for fresh [releases](https://bakery.cakephp.org/categories/release.html).
+CakePHP follows [Semantic Versioning](https://book.cakephp.org/4/en/release-policy.html). And, from
+3.x onward, major versions also have a name:
```

Comment:
```suggestion
CakePHP follows [Semantic Versioning](https://book.cakephp.org/4/en/release-policy.html). 
From 3.x onward, major versions also have a name:
```

I've learned once that you shall not start a sentence with "and", at least not in German. Not sure if this applies to English, too.
It's two enough distinct things, also.

Reply:
Don't know about the rule in english, but your suggestion looks better anyway. The `also` can also be removed I think.